src/Mailman.Server/Controllers/MergeTemplatesController.cs

```diff
@@ -20,13 +22,16 @@ namespace Mailman.Server.Controllers
     /// <summary>
     /// Controller for Merge Templates.
     /// </summary>
-    [Authorize]
+    [Authorize(AuthenticationSchemes = AuthSchemes)]
```

Comment:
I didn't know you could reference private members inside the class!
Good to know.